test/e2e/framework/test_context.go

```diff
@@ -321,6 +322,22 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 	flags.IntVar(&TestContext.SnapshotControllerHTTPPort, "snapshot-controller-http-port", 0, "The port to use for snapshot controller HTTP communication.")
 }
 
+func SetGinkgoConfig() (types.SuiteConfig, types.ReporterConfig) {
```

Comment:
I find the name misleading because the functions doesn't actually "set" the configuration. How about `CreateGinkgoConfig` because that is what it does? The caller then needs to use it.

Reply:
`CreateGinkgoConfig` looks better, I will updated this later, let me hold for an while until all jobs are clear.

Reply:
Done